lib/middleware.js

```diff
@@ -14,6 +14,7 @@ module.exports = function wrapper(context) {
 
       return new Promise(((resolve) => {
         ready(context, () => {
+          res.locals = res.locals || {};
```

Comment:
Can you link to any issue regarding this or fill a longer description of what you're trying to fix? Would help us understand your problem space, as this issue might be deeper than just setting a default. Thanks! ❤️